mscore/scoreview.cpp

```diff
@@ -141,7 +141,18 @@ ScoreView::ScoreView(QWidget* parent)
       double mag  = preferences.getDouble(PREF_SCORE_MAGNIFICATION) * (mscore->physicalDotsPerInch() / DPI);
       _matrix     = QTransform(mag, 0.0, 0.0, mag, 0.0, 0.0);
       imatrix     = _matrix.inverted();
-      _magIdx     = preferences.getDouble(PREF_SCORE_MAGNIFICATION) == 1.0 ? MagIdx::MAG_100 : MagIdx::MAG_FREE;
+      switch (preferences.getInt(PREF_SCORE_ZOOM_TYPE)) {
+            case 0: // Percentage
+                  _magIdx = preferences.getDouble(PREF_SCORE_MAGNIFICATION) == 1.0 ? MagIdx::MAG_100 : MagIdx::MAG_FREE;
+            case 1: // Page Width
```

Comment:
Or a  `Q_FALLTHROUGH();`  or a `// FALLTHROUGH`, it that is the intention. It doesn't seem to be though

Reply:
Yeah, the enumeration would make the code read better. Will think about doing it after cleaning up some midnight problems and hoping the 100% zoom fixes the problem. In the mean time, I'll switch the integer id's so that the uppermost value falls through with the default label.

Reply:
Still, I find it slightly fascinating that not being at 100% zoom causes a problem with the testing scripts.